images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -141,6 +141,10 @@ function Get-AzureCLIVersion {
     return "Azure CLI $azureCLIVersion"
 }
 
+function Get-AzCopyVersion {
+    return ($(azcopy --version) -replace "version ")
```

Comment:
nitpick: Do we really need return and brackets?  `(azcopy --version).replace('version ','')`

Reply:
"return" is present in all functions in SoftwareReport.Tools.psm1 module.
I've decided to use the same style